src/constants.ts

```diff
@@ -216,3 +216,9 @@ export const RELEASE_NOTES_BASE_URL = 'https://github.com/ethereum/go-ethereum/r
 
 // Code snippet class constants
 export const CLASSNAME_PREFIX = 'language-';
+
+// Amount of releases per os constants
+export const AMOUNT_OF_RELEASES_PER_DARWIN = 2;
+export const AMOUNT_OF_RELEASES_PER_LINUX = 12;
```

Comment:
found that the amount of releases in linux is not always 12, in some older releases is 18 or 20 (check on https://geth.ethereum.org/downloads/).

between **1.10.9** and **1.9.25** included, is 18 (screenshot below)

<img width="681" alt="Screen Shot 2022-12-14 at 09 43 54" src="https://user-images.githubusercontent.com/948922/207601118-397bdb3a-94a7-433c-a091-6a7c13ad7c91.png">

**1.9.24** and older is 20 (screenshot below)

<img width="690" alt="Screen Shot 2022-12-14 at 09 47 18" src="https://user-images.githubusercontent.com/948922/207601246-01222119-35dd-4971-b104-d0ca09f85c40.png">

This only apply to Linux releases, in the other cases the amount is always the same fixed vlue.